roles/nginx/molecule/default/molecule.yml

```diff
@@ -4,7 +4,7 @@ dependency:
 driver:
   name: docker
 platforms:
-  - name: nginx
+  - name: nginx1
```

Comment:
is this required?

Reply:
we just need to decide a convention because I've started to use `role-instance` elsewhere